### PR TITLE
[2.5] Add NGINX v1.1.0

### PR DESCRIPTION
**Pull request by rayandas**
Issue: https://github.com/rancher/rancher/issues/36136

### CODE REVIEW

File: rke/k8s_rke_system_images.go [hunk 1]
```diff
@@ -5563,6 +5563,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/kubelet-pause:v0.1.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.13",
 		},
+		// Enabled in 2.5.12
+		"v1.19.16-rancher1-3": {
```
Comment (a-blender) [inline rke/k8s_rke_system_images.go:5567]: Still new to this, but you might want to put a comment for whether Kubernetes hyperkube was updated (it should have been I think) and what other image versions changed. I just know that would make surfing through this file a lot easier for other devs

File: rke/k8s_rke_system_images.go [hunk 2]
```diff
@@ -6086,6 +6130,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/kubelet-pause:v0.1.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.13",
 		},
+		// Enabled in v2.5.12
+		"v1.20.14-rancher2-2": {
```
Comment (a-blender) [inline rke/k8s_rke_system_images.go:6134]: Same here

File: rke/templates/templates.go [hunk 3]
```diff
@@ -151,12 +152,14 @@ func LoadK8sVersionedTemplates() map[string]map[string]string {
...
-			">=1.20.0-rancher0 <1.20.12-rancher1-1":   nginxIngressV11512,
-			">=1.20.12-rancher1-1":                    nginxIngressv0493,
+			">=1.15.12-rancher1-1 <1.16.1-rancher1-1":  nginxIngressV11512,
+			">=1.16.10-rancher1-1 <1.17.0-rancher1-1":  nginxIngressV11512,
+			">=1.17.6-rancher1-1 <1.19.16-rancher1-1":  nginxIngressV11512,
+			">=1.19.16-rancher1-1 <1.19.16-rancher1-3": nginxIngressv0493,
+			">=1.19.16-rancher1-3 <1.20.0-rancher0":    nginxIngressv110,
+			">=1.20.0-rancher0 <1.20.12-rancher1-1":    nginxIngressV11512,
+			">=1.20.12-rancher1-1 <1.20.14-rancher2-2": nginxIngressv0493,
+			">=1.20.14-rancher2-2":                     nginxIngressv110,
```
Comment (a-blender) [inline rke/templates/templates.go:162]: LGTM

**Review by a-blender — APPROVED** [review]

**Comment by kinarashah** [comment]
@rayandas we won't be able to use the template directly for v2.5. I opened https://github.com/rancher/kontainer-driver-metadata/pull/815 to address the issue, please let me know if you have questions on this.

**Comment by rayandas** [comment]
Ah! I got the `DefaultIngresClass` issue. Closing this PR.

**Pull request closed without merging**